### DISCUSSION TITLE
iOS Shortcut for share sheet

### DISCUSSION THREAD

**Discussion by joestump** [JoTec2002/InstagramToMealie#4] [General]
I've created an iOS Shortcut that adds an "Add to Mealie" shortcut to share sheets. If the URL contains `instagram.com`, it will forward it to the `InstagramToMealie` instance. If not, it will send it to Mealie using a `Bearer` API token. 

You set up three variables on install:

* InstagramToMealie URL
* Mealie URL
* Mealie API token

https://www.icloud.com/shortcuts/4728f51e401f4901ad74748d6157fbb3

**Edit:** Updated iCloud URL.

**Comment by philippbla** [comment]
I gave it a shot myself and included some user feedback in case the submission was not successful. I'm sure it could be improved further.
![screenshot](https://github.com/user-attachments/assets/7478433e-d07b-48e2-bcd2-2c490e9e0314)
